_data/releases.yml

```diff
@@ -4188,7 +4188,7 @@
     docker_arm: true
   source: true
   previous_release: v22.2.7
-  
```

Comment:
@mdlinville If you want to test the latest batch of changes, make sure you add `linux_intel_fips: true` to one or more releases, and then stop and start Jekyll.